task.sql

```diff
@@ -3,31 +1,55 @@
 CREATE DATABASE ShopDB;
 USE ShopDB;
 
 CREATE TABLE Countries (
-    ID INT,
-    Name VARCHAR(50),
-    PRIMARY KEY (ID)
+  ID INT,
+  Name VARCHAR(50),
+  
+  PRIMARY KEY (ID)
+);
+
+CREATE TABLE Warehouses (
+  ID INT,
+  Name VARCHAR(50),
+  Address VARCHAR(50),
+  CountryID INT,
+
+  PRIMARY KEY (ID),
+  FOREIGN KEY (CountryID) REFERENCES Countries(ID) ON DELETE NO ACTION
```

Comment:
The `ON DELETE NO ACTION` constraint means that you cannot delete a country if there are warehouses associated with it. Ensure this is the desired behavior. If you want to allow deletion and automatically remove associated warehouses, consider using `ON DELETE CASCADE`.

Reply:
Fixed